### PR TITLE
docs(client): expanded info on client installation docs

### PR DESCRIPTION
**Pull request by jackalexanderhippo**

### CODE REVIEW

**Review by vickybirtleshippo — APPROVED** [review]

File: docs/install.md [hunk 1]
```diff
@@ -38,17 +38,19 @@ The client requires the Conda platform to be using `x86_64` when creating the en
     ```
 
 ### Installing or updating the client with Miniconda
+The client has at least one dependency that requires `bioconda`, which itself
+depends on `conda-forge`. Note that for the `conda create` step (see below), installation can be very slow,
+so please leave it running. For more verbose output, you can add the `-v` or `-vv` flags, though
+it is not recommended to show the full debug output with `-vvv` as this has been seen to lead to OOM errors.
```
Comment (JDhillonEIT) [inline docs/install.md:44]:
```suggestion
### Installing or updating the client with Miniconda

The client has at least one dependency that requires `bioconda`, which itself
depends on `conda-forge`. Note that for the `conda create` step (see below), installation can be very slow,
so please leave it running. For more verbose output, you can add the `-v` or `-vv` flags, though
it is not recommended to show the full debug output with `-vvv` as this has been seen to lead to OOM errors.
```

File: docs/install-dev.md [hunk 2]
```diff
@@ -2,9 +2,12 @@
 
 ## Development install
 
+Installation of the client requires the `conda` package manager, as well as
+`conda-forge` and `bioconda`, in order to install the required libraries. For more
+information on how to install that, please refer to [this section](install.md#installing-miniconda).
 ```bash
 git clone https://github.com/EIT-Pathogena/client.git
```
Comment (JDhillonEIT) [inline docs/install-dev.md:9]:
```suggestion
Installation of the client requires the `conda` package manager, as well as
`conda-forge` and `bioconda`, in order to install the required libraries. For more
information on how to install that, please refer to [this section](install.md#installing-miniconda).

```bash
git clone https://github.com/EIT-Pathogena/client.git
```

File: docs/install.md (hunk 1 above)
Comment (JDhillonEIT) [reply docs/install.md:44]: I'm a Markdown pedant

**Review by JDhillonEIT — COMMENTED** [review]
Very minor pedantry from me!

File: docs/install.md (hunk 1 above)
Comment (jackalexanderhippo) [reply docs/install.md:44]: I didn't know spacing below titles was a thing - better add back in those that I've removed later in the file too then.

**Review by vickybirtleshippo — APPROVED** [review]

**Pull request merged**